src/components/ForgotPassword/ForgotPassword.jsx

```diff
@@ -1,29 +1,155 @@
 import React, { useState } from 'react';
-import { Button } from 'react-bootstrap';
-import styled from 'styled-components';
-
-const SubmitButton = styled(Button)`
-    background-color: #6593D6;
-    color: white;
-    padding: 10px;
-    border-radius: 10px
-    border: none;
-`;
-
-export default () => {
-  const [email, setEmail] = useState('');
-
-  const checkEmail = () => {
-    if (email === '') {
-      window.alert('Enter an email');
+import { Auth } from 'aws-amplify';
+import { Link } from 'react-router-dom';
+import {
+  FormGroup,
+  FormControl,
+  FormLabel,
+} from 'react-bootstrap';
+
+export default function ResetPassword() {
+  const [code, setCode] = useState('');
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+
+  const [codeSent, setCodeSent] = useState(false);
+  const [confirmed, setConfirmed] = useState(false);
+  const [isConfirming, setIsConfirming] = useState(false);
+  const [isSendingCode, setIsSendingCode] = useState(false);
+
+  function validateCodeForm() {
+    return username.length > 0;
+  }
```

Comment:
You don't need a separate function for this. In JS some values are considered "falsey" meaning while they aren't strictly "false" they are interpreted as such. Examples of these are 0, '', and null. If you're checking if username.length > 0 you can just say if (username.length). If username.length does not equal 0, it will return true.

Reply:
https://developer.mozilla.org/en-US/docs/Glossary/Falsy